admin/services/date.service.js

```diff
@@ -153,6 +153,20 @@ const dateService = {
 
   formatFileName: function (date) {
     return moment(date).format(filenameFriendly)
+  },
+
+  /**
+   * Checks if a moment date is between a given timeframe with additional parameters
+   * @param {object} date
+   * @param {object} startDate
+   * @param {object} endDate
+   * @param {null|string} granularity
+   * @param {boolean} isInclusive
+   * @returns {boolean}
+   */
+  isBetween: function (date, startDate, endDate, granularity = null, isInclusive = false) {
```

Comment:
This is workable code, but passing the boolean for `isInclusive` as the 5th parameter is asking the reader to do a lot of work.

I would name the function `isBetweenInclusive()` and not have the 5th parameter.  

Alternatively, instead of passing a raw boolean primitive, make the code more readable:

```
const inclusive = true
...
if (dateService.isBetween(dt, activeCheckWindow.adminStartDate, activeCheckWindow.adminEndDate, null, inclusive)) {
```

Reply:
@jon-shipley I favoured the  `isBetweenInclusive`. Done